core/src/main/scala/kafka/server/MetadataCache.scala

```diff
@@ -155,7 +155,13 @@ private[server] class MetadataCache(brokerId: Int) extends Logging {
 
   def getControllerId: Option[Int] = controllerId
 
-  def updateCache(correlationId: Int, updateMetadataRequest: UpdateMetadataRequest) {
+  /**
+    *
```

Comment:
nit: unless we're going to fill out the the description and parameters, maybe we shouldn't make this a scaladoc?